<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
"http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head>
<title>Unexpected I/O log</title>
<meta http-equiv="cache-control" content="no-cache"></meta>
<meta http-equiv="content-type" content="text/html; charset=utf-8"></meta>
<link rel="stylesheet" href="../../ct_default.css" type="text/css"></link>
<script type="text/javascript" src="../../jquery-latest.js"></script>
<script type="text/javascript" src="../../jquery.tablesorter.min.js"></script>
</head>
<body>

<br />
<h3>Unexpected I/O</h3>
<pre>

</pre>
<div class="error_logger"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-14 20:08:13.027 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:08:13.026832 ===
** Generic server my_test_name terminating 
** Last message in was stop
** When Server state == {formatted,[]}
** Reason for termination ==
** stopped
** Client <0.247083.0> stacktrace
** [{gen_server,mc_recv,5,[{file,"gen_server.erl"},{line,1975}]},
    {gen_server,multi_call,4,[{file,"gen_server.erl"},{line,1936}]},
    {gen_server_SUITE,multicall,1,[{file,"gen_server_SUITE.erl"},{line,1487}]},
    {test_server,ts_tc,3,[{file,"test_server.erl"},{line,1794}]},
    {test_server,run_test_case_eval1,6,[{file,"test_server.erl"},{line,1303}]},
    {test_server,run_test_case_eval,9,[{file,"test_server.erl"},{line,1235}]}]


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during multi_call_parallel in gen_server_SUITE 2025-01-14 20:08:13.027 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:08:13.027241 ===
  crasher:
    initial call: gen_server_SUITE:init/1
    pid: <0.247091.0>
    registered_name: my_test_name
    exception exit: stopped
      in function  gen_server:handle_msg/6 (gen_server.erl, line 2428)
    ancestors: [<0.247083.0>]
    message_queue_len: 0
    messages: []
    links: [<0.247083.0>]
    dictionary: []
    trap_exit: false
    status: running
    heap_size: 987
    stack_size: 29
    reductions: 594
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-14 20:08:53.444 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:08:53.444361 ===
** Generic server hurra terminating 
** Last message in was {'EXIT',&lt;0.251436.0&gt;,
                               {#Ref&lt;0.2256082971.3766484994.146759&gt;,1003907,
                                ok,
                                [{gen_server_SUITE,spec_init}],
                                []}}
** When Server state == {formatted,{}}
** Reason for termination ==
** {#Ref&lt;0.2256082971.3766484994.146759&gt;,1003907,ok,
    [{gen_server_SUITE,spec_init}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during gen_server_SUITE:spec_init/1 2025-01-14 20:08:53.444 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:08:53.444627 ===
  crasher:
    initial call: gen_server_SUITE:spec_init_global_default_timeout/2
    pid: &lt;0.251457.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2256082971.3766484994.146759&gt;,1003907,ok,
                     [{gen_server_SUITE,spec_init}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [&lt;0.251436.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 414
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during log_mf_h_SUITE:test/1 2025-01-14 20:13:36.309 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:13:36.309245 ===
  crasher:
    initial call: gen_event:init_it/6
    pid: &lt;0.257268.0&gt;
    registered_name: []
    exception exit: {#Ref&lt;0.2256082971.3774087169.84273&gt;,5469,
                     {ok,3},
                     [{log_mf_h_SUITE,test}],
                     []}
      in function  gen_event:terminate_server/4 (gen_event.erl, line 1696)
    ancestors: [&lt;0.257266.0&gt;]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 6772
    stack_size: 29
    reductions: 28121
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-14 20:13:45.880 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:13:45.880459 ===
** Generic server <0.258554.0> terminating 
** Last message in was {nodedown,'peer_down_crash-556610-11901@858101ec0371'}
** When Server state == {peer_state,
                            #{args => ["-connect_all","false"],
                              name => "peer_down_crash-556610-11901",
                              peer_down => crash},
                            'peer_down_crash-556610-11901@858101ec0371',
                            [47,98,117,105,108,100,114,111,111,116,47,111,116,
                             112,47,69,114,108,97,110,103,32,8709,8868,8478,
                             47,101,114,116,115,45,49,53,46,50,47,98,105,110,
                             47,101,114,108],
                            ["-sname","peer_down_crash-556610-11901",
                             "-connect_all","false","-detached",
                             "-peer_detached","-user","peer","-origin",
                             "g1h3GHRlc3Rfc2VydmVyQDg1ODEwMWVjMDM3MQAD8foAAAAAZ4azdg=="],
                            undefined,undefined,<<>>,
                            {down,
                                {nodedown,
                                    'peer_down_crash-556610-11901@858101ec0371'}},
                            {<0.258538.0>,#Ref<0.2256082971.3774087169.98022>},
                            0,#{}}
** Reason for termination ==
** {nodedown,'peer_down_crash-556610-11901@858101ec0371'}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during dist in peer_SUITE 2025-01-14 20:13:45.881 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:13:45.880836 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258554.0>
    registered_name: []
    exception exit: {nodedown,'peer_down_crash-556610-11901@858101ec0371'}
      in function  gen_server:handle_common_reply/8 (gen_server.erl, line 2478)
    ancestors: [<0.258538.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1483
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-14 20:13:46.778 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:13:45.776704 ===
Process <0.92.0> on node 'peer_SUITE-shutdown_stop-432068-11901@858101ec0371' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-14 20:13:47.342 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:13:47.342739 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-411107-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during dist in peer_SUITE 2025-01-14 20:13:47.411 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:13:47.411723 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-432100-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-14 20:14:03.043 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:02.042117 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-354533-11901@858101ec0371' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-14 20:14:03.586 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:03.586729 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-354501-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during tcp in peer_SUITE 2025-01-14 20:14:03.597 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:03.597690 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-432356-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:05.749 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:14:05.749567 ===
** Generic server <0.258918.0> terminating 
** Last message in was {'EXIT',<0.258914.0>,
                               {#Ref<0.2256082971.3766222851.240573>,1614673,
                                ok,
                                [{peer_SUITE,cntrl_channel_handler_crash}],
                                []}}
** When Server state == {peer_state,#{name =>
                                          "peer_SUITE-cntrl_channel_handler_crash-355045-11901",
                                      connection => standard_io},
                                    'peer_SUITE-cntrl_channel_handler_crash-355045-11901@858101ec0371',
                                    [47,98,117,105,108,100,114,111,111,116,47,
                                     111,116,112,47,69,114,108,97,110,103,32,
                                     8709,8868,8478,47,101,114,116,115,45,49,
                                     53,46,50,47,98,105,110,47,101,114,108],
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash-355045-11901",
                                     "-user","peer"],
                                    #Port<0.3135>,undefined,<<>>,running,
                                    {<0.258914.0>,
                                     #Ref<0.2256082971.3766484996.102458>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2256082971.3766222851.240573>,1614673,ok,
    [{peer_SUITE,cntrl_channel_handler_crash}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:05.750 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:14:05.749883 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258918.0>
    registered_name: []
    exception exit: {#Ref<0.2256082971.3766222851.240573>,1614673,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258914.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3135>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4630
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:06.340 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:14:06.339551 ===
** Generic server <0.258939.0> terminating 
** Last message in was {'EXIT',<0.258882.0>,
                           {#Ref<0.2256082971.3766747138.185576>,2208151,ok,
                            [{peer_SUITE,
                                 cntrl_channel_handler_crash_old_release}],
                            []}}
** When Server state == {peer_state,#{args =>
                                          ["-setcookie",
                                           "PQRYZJXRKCYNPGUZTWCU","-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                           "-env","ERL_CRASH_DUMP",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@858101ec0371.2025-01-14_19.56.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-354917-11901",
                                           "-pa",
                                           "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@858101ec0371.2025-01-14_19.56.54/make_test_dir.stdlib_test.logs/run.2025-01-14_19.57.30/log_private/"],
                                      env => [{"ERL_AFLAGS",false}],
                                      name =>
                                          "peer_SUITE-cntrl_channel_handler_crash_old_release-354917-11901",
                                      connection => standard_io,
                                      exec =>
                                          "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                      start_cover => false},
                                    'peer_SUITE-cntrl_channel_handler_crash_old_release-354917-11901@858101ec0371',
                                    "/usr/local/lib/erlang-26.2.5.6/bin/erl",
                                    ["-sname",
                                     "peer_SUITE-cntrl_channel_handler_crash_old_release-354917-11901",
                                     "-setcookie","PQRYZJXRKCYNPGUZTWCU",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/stdlib_test",
                                     "-env","ERL_CRASH_DUMP",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@858101ec0371.2025-01-14_19.56.54/erl_crash_dump.peer_SUITE-cntrl_channel_handler_crash_old_release-354917-11901",
                                     "-pa",
                                     "/buildroot/otp/lib/stdlib/make_test_dir/ct_logs/ct_run.test_server@858101ec0371.2025-01-14_19.56.54/make_test_dir.stdlib_test.logs/run.2025-01-14_19.57.30/log_private/",
                                     "-user","peer"],
                                    #Port<0.3156>,undefined,<<>>,running,
                                    {<0.258882.0>,
                                     #Ref<0.2256082971.3766747138.186346>},
                                    0,#{}}
** Reason for termination ==
** {#Ref<0.2256082971.3766747138.185576>,2208151,ok,
    [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:06.340 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:14:06.340131 ===
  crasher:
    initial call: peer:init/1
    pid: <0.258939.0>
    registered_name: []
    exception exit: {#Ref<0.2256082971.3766747138.185576>,2208151,ok,
                     [{peer_SUITE,cntrl_channel_handler_crash_old_release}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.258882.0>]
    message_queue_len: 1
    messages: [{'EXIT',#Port<0.3156>,normal}]
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 2586
    stack_size: 29
    reductions: 4250
  neighbours:


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:06.718 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:06.717878 ===
peer:stop() timed out waiting for disconnect from node 'init_debug-432740-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:06.792 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:05.790495 ===
Process <0.89.0> on node 'peer_SUITE-shutdown_stop-354981-11901@858101ec0371' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:07.022 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:07.022815 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_stop_timeout-411875-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during standard_io in peer_SUITE 2025-01-14 20:14:07.180 ***</b>
=WARNING REPORT==== 14-Jan-2025::20:14:07.180688 ===
peer:stop() timed out waiting for disconnect from node 'peer_SUITE-shutdown_halt_timeout-411939-11901@858101ec0371'. The connection was forcefully taken down.

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-14 20:36:00.001 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:36:00.001172 ===
** Generic server <0.335832.0> terminating 
** Last message in was {'EXIT',<0.335829.0>,
                               {#Ref<0.2256082971.3782475778.2715>,350611,
                                [{child1,<0.335838.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335832.0>,supervisor_SUITE},
                            rest_for_one,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335838.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458407],
                            1,0,never,supervisor_SUITE,
                            {ok,{{rest_for_one,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2256082971.3782475778.2715>,350611,
    [{child1,<0.335838.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-14 20:36:00.001 ***</b>
=ERROR REPORT==== 14-Jan-2025::20:36:00.001180 ===
** Generic server <0.335831.0> terminating 
** Last message in was {'EXIT',<0.335829.0>,
                               {#Ref<0.2256082971.3782475778.2715>,350611,
                                [{child1,<0.335838.0>,worker,[]}],
                                [{supervisor_SUITE,temporary_bystander}],
                                []}}
** When Server state == {state,
                            {<0.335831.0>,supervisor_SUITE},
                            one_for_all,
                            {[child1],
                             #{child1 =>
                                   {child,<0.335837.0>,child1,
                                       {supervisor_1,start_child,[]},
                                       permanent,false,100,worker,[]}}},
                            undefined,2,300,
                            [-576458407],
                            1,0,never,supervisor_SUITE,
                            {ok,{{one_for_all,2,300},[]}}}
** Reason for termination ==
** {#Ref<0.2256082971.3782475778.2715>,350611,
    [{child1,<0.335838.0>,worker,[]}],
    [{supervisor_SUITE,temporary_bystander}],
    []}


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-14 20:36:00.001 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:36:00.001592 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335832.0>
    registered_name: []
    exception exit: {#Ref<0.2256082971.3782475778.2715>,350611,
                     [{child1,<0.335838.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335829.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1027
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_SUITE:temporary_bystander/1 2025-01-14 20:36:00.002 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:36:00.001691 ===
  crasher:
    initial call: supervisor:supervisor_SUITE/1
    pid: <0.335831.0>
    registered_name: []
    exception exit: {#Ref<0.2256082971.3782475778.2715>,350611,
                     [{child1,<0.335838.0>,worker,[]}],
                     [{supervisor_SUITE,temporary_bystander}],
                     []}
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [<0.335829.0>]
    message_queue_len: 0
    messages: []
    links: []
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 1598
    stack_size: 29
    reductions: 1009
  neighbours:


</pre></div>
<pre>
</pre>
<div class="sasl"><pre><b>*** System report during supervisor_bridge_SUITE:simple_global_supervisor/1 2025-01-14 20:36:43.514 ***</b>
=CRASH REPORT==== 14-Jan-2025::20:36:43.513888 ===
  crasher:
    initial call: supervisor_bridge:supervisor_bridge_SUITE/1
    pid: <0.436001.0>
    registered_name: []
    exception exit: killed
      in function  gen_server:decode_msg/9 (gen_server.erl, line 2301)
    ancestors: [bridge9212,<0.435996.0>]
    message_queue_len: 0
    messages: []
    links: [<0.436002.0>]
    dictionary: []
    trap_exit: true
    status: running
    heap_size: 610
    stack_size: 29
    reductions: 388
  neighbours:
    neighbour:
      pid: <0.436002.0>
      registered_name: work_supervisor_bridge_SUITE_server
      initial_call: {supervisor_bridge_SUITE,internal_loop_init,1}
      current_function: {supervisor_bridge_SUITE,internal_loop,1}
      ancestors: []
      message_queue_len: 0
      links: [<0.436001.0>]
      trap_exit: false
      status: waiting
      heap_size: 233
      stack_size: 3
      reductions: 9
      current_stacktrace: [{supervisor_bridge_SUITE,internal_loop,1,
                      [{file,"supervisor_bridge_SUITE.erl"},{line,152}]}]

</pre></div>
<pre>
</pre>
<div class="error_logger"><pre><b>*** System report during zzz_SUITE:lc_graph/1 2025-01-14 21:02:53.354 ***</b>
=WARNING REPORT==== 14-Jan-2025::21:02:53.351968 ===
Process <0.439501.0> on node 'test_server@858101ec0371' enabled access to the emulator internal state.
NOTE: This is an erts internal test feature and should *only* be used by OTP test-suites.



</pre></div>
<pre>
</pre>
<center>
<br /><hr /><p>
<a href="../../../all_runs.html">Test run history
</a>  |  <a href="../../../index.html">Top level test index
</a>  |  <a href="../../../suite.log.latest.html">Latest test result</a>
</p>
<div class="copyright">Copyright &copy; 2025 <a href="http://www.erlang.org">Open Telecom Platform</a><br />
Updated: <!--date-->Tue Jan 14 2025 19:57:30<!--/date--><br />
</div>
</center>
</body>
</html>
